<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(178, 172, 136);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Savings</string>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fund_add">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(79, 52, 0,179);</string>
    </property>
    <property name="text">
     <string>Add Funds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fund_remove">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(79, 52, 0,179);</string>
    </property>
    <property name="text">
     <string>Remove Funds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_amount">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="amount_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Amount:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Menlo</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(79, 52, 0,179);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fund_add</sender>
   <signal>clicked()</signal>
   <receiver>finish_button</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fund_add</sender>
   <signal>clicked()</signal>
   <receiver>input_amount</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fund_add</sender>
   <signal>clicked()</signal>
   <receiver>amount_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
